devtools_m2/cypress/support/docker_helpers.js

```diff
@@ -5,5 +5,5 @@ Cypress.Commands.add("flushCaches", () => {
 
 beforeEach(function() {
   cy.log('resetting docker for test')
-  cy.exec('DRIP_COMPOSE_ENV=test ./reset.sh')
+  cy.exec('DRIP_COMPOSE_ENV=test ./reset.sh', {timeout: 120000})
```

Comment:
While this is typically executing within 20 seconds on my machine, sometimes it takes quite awhile to run, and I have no idea why.  However, this seems to be long enough to let it complete.

Reply:
Might be worth annotating the reset file to have better logs…

Reply:
The reset file itself is pretty simple (i.e. there's not a lot to annotate).  The long execution is in the two calls to Magento, not the DB.